doc/api/debugger.md

```diff
@@ -167,6 +167,10 @@ breakpoint)
 
 ## Advanced Usage
 
+### TCP-based protocol
+
+> Stability: 0 - Deprecated: Use [V8 Inspector Integration][] instead. The debug protocol used by the `--debug` flag was removed from V8.
```

Comment:
nit: Can you wrap this at 80 characters? You should be able to use multiple `>`s to continue the blockquote, the documentation generator should be able to handle that